_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -233,8 +233,8 @@ It finds its configured name `sidecar.initializer.istio.io`
 as the first in the list of pending initializers.
 
 3) istio-initializer checks to see if it was responsible for
-initializing the namespace of the workload. No further work is done
-and the initializer ignores the workload if the initializer is
+initializing workloads in the namespace of the workload. No further work is done
```

Comment:
double workload ?

Reply:
It was intentional, but in hindsight it sounds less awkward to say "initializing workloads in its namespace." Will do!

Reply:
Actually, then again, "its" is unclear, since "it" could refer to the `istio-initializer`, which makes no sense. Perhaps, "initializing workloads in the namespace of the uninitialized workload."?

Reply:
maybe
"istio-initializer checks to see it is configured to manage the namespace of the new workload." ...

but it's fine as is when reading it all at once, thanks